packages/markdown-docx/src/ToOOXMLVisitor/index.js

```diff
@@ -72,11 +76,15 @@ class ToOOXMLVisitor {
             for (let subNode of node) {
                 if (this.getClass(subNode) === TRANSFORMED_NODES.text) {
                     let propertyTag = '';
+                    let isLinkPropertyPresent = false;
                     for (let property of properties) {
                         if (property === TRANSFORMED_NODES.emphasize) {
                             propertyTag += EMPHASIS_RULE();
                         } else if (property === TRANSFORMED_NODES.strong) {
                             propertyTag += STRONG_RULE();
+                        } else if (property === TRANSFORMED_NODES.link) {
+                            isLinkPropertyPresent = true;
+                            propertyTag+= LINK_PROPERTY_RULE();
```

Comment:
Your formatter didn't add space between `propertyTag` and `+=`.

Reply:
@algomaster99 
Will update this.